teach.md

```diff
@@ -0,0 +1,217 @@
+prematurely and get a feeling of bad self-efficacy and loose motivation.
+
+> “In my experience with Scala, the dialog with the compiler about static
+> errors strongly supports conceptual learning and strengthens self-efficacy:
+> 'I can do this and I’m getting a grip of it!’.”
+> 
+> Björn Regnell, Lund University, Sweden.
+
+### Regularity and Principled Constructs
+
+Scala has done away with many of the quirks from C-like languages that still
+remain in e.g. Java and C#. There are, for instance, no surprising restrictions
+on what can be nested inside what, and no irregular semantic difference between
+primitive non-objects and “real” objects.
+
+Practices recognized by experts as best practices (e.g.,
+[avoiding the usage of `null`](https://www.infoq.com/presentations/Null-References-The-Billion-Dollar-Mistake-Tony-Hoare/),
+[manipulating immutable values](https://www.infoq.com/articles/data-oriented-programming-java/))
+have been the norm since the inception of Scala.
+
+> “The level of regularity and generality of principled concepts in Scala is
+> the best I have seen so far in a multi-paradigm language. Every strange
+> irregularity and tricky exception wear down the grit of students and take
+> up time in teaching that can be much better spent.”
+> 
+> Björn Regnell, Lund University, Sweden.
+
+> “ After learning Scala, many students don’t want to ever go back to more
+> primitive and ad-hoc languages like Java and C++, unless forced by the
+> curriculum!”
+> 
+> Lionel Parreaux, Hong Kong University of Science and Technology, Hong Kong.
+
+### Adoption in the Industry
+
+Scala is used by companies such as Twitter, Netflix, AirBnB, NASA, Lego, 
+Spotify (to name a few).
+
+It is one of the highest-ranked languages in the [Redmonk Programming Language
+Ranking](https://redmonk.com/data/).
+
+### Multiplatform
+
+Scala has been successfully used to implement both backends (running on a JVM)
+and frontends (running in the browser).
+
+### Ecosystem
+
+Scala has [thousands of libraries](https://index.scala-lang.org/awesome), and
+it has access to the whole Java ecosystem as well as the JavaScript ecosystem.
+
+> “Full access to Java libraries as well as a large environment of Scala
+> libraries means that I can give interesting assignments.”
+> 
+> Mark Lewis, Trinity University, USA.
+
+### Versatile
+
+Scala has been successfully used in domains such as distributed services, 
+parallel programming, big data analysis, HTTP servers.
+
+> “In CS2, I’ve had students do a project that involves graphics, networking,
+> and multithreading and Scala makes that possible with limited overhead.”
+> 
+> Mark Lewis, Trinity University, USA.
+
+## Who Is Teaching Scala
+
+Several universities use Scala to teach programming. Discover them on the
+following map:
+
+{% include teachers-map.html %}
+
+Want to add yourself to the map? Add your name
+[here](https://github.com/scala/docs.scala-lang/edit/main/_data/teachers.yml).
```

Comment:
Will this page move to [scala/docs.scala-lang](https://github.com/scala/docs.scala-lang)?

Reply:
Good catch! Fixed in 55dfe20f